docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -791,6 +792,43 @@ PanacheQuery<DogDto> query = Dog.findAll().project(DogDto.class);
 ----
 <1> The `ownerName` DTO constructor's parameter will be loaded from the `owner.name` HQL property.
 
+It's also possible to specify a HQL query with a select clause. In this case, the projection class must have a constructor
```

Comment:
I think we try to avoid `it's` in docs (not saying we don't have many occurrences but let's try to not introduce new ones).

```suggestion
It is also possible to specify a HQL query with a select clause. In this case, the projection class must have a constructor
```